apps/hash-graph/postgres_migrations/V8__ontology_create_functions.sql

```diff
@@ -0,0 +1,159 @@
+DROP FUNCTION
+  create_ontology_id (
+    "ontology_id" UUID,
+    "base_url" TEXT,
+    "version" BIGINT,
+    "record_created_by_id" UUID
+  );
+
+DROP FUNCTION
+  create_owned_ontology_id (
+    "base_url" TEXT,
+    "version" BIGINT,
+    "record_created_by_id" UUID,
+    "owned_by_id" UUID
+  );
+
+DROP FUNCTION
+  create_external_ontology_id (
+    "base_url" TEXT,
+    "version" BIGINT,
+    "record_created_by_id" UUID,
+    "fetched_at" TIMESTAMP WITH TIME ZONE
+  );
+
+CREATE FUNCTION
+  create_ontology_id (
+    "base_url" TEXT,
+    "version" BIGINT,
+    "record_created_by_id" UUID,
+    "resume_on_conflict" BOOLEAN,
+    "is_external" BOOLEAN
+  ) RETURNS TABLE (ontology_id UUID) AS $create_owned_ontology_id$
```

Comment:
Do you want to rename `create_owned_ontology_id` to `create_ontology_id`?

Reply:
Thanks, fixed it in a16c09b152ec54bfdc80a9ce2763aeab4d9553c1 (but GitHub does not show the commit...)